system/Debug/Exceptions.php

```diff
@@ -330,7 +339,7 @@ protected function collectVars(Throwable $exception, int $statusCode): array
 	 *
 	 * @param \Throwable $exception
 	 *
-	 * @return array
+	 * @return integer[]
```

Comment:
Is this actually a valid docblock return type?

Reply:
This was actually `int[]` but phpcbf changed it to the long form because `codeigniter4/coding-standard` changes `bool` and `int` to their respective long forms.

Reply:
Fixed docblock